app.rb

```diff
@@ -0,0 +1,95 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './classroom'
+require_relative './rental'
+
+class App
+  attr_accessor :books, :students, :teachers, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def display_all_books
+    @books.each_with_index do |book, index|
+      puts "#{index}) Title: '#{book.title}', Author: '#{book.author}'"
+    end
+  end
+
+  def display_all_people
+    @people.each_with_index do |people, index|
+      puts "#{index}) Name: #{people.name}, ID: #{people.id}, Age: #{people.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student(1) or a teacher(2)? [Input the number]: '
+    person_option = gets.chomp.to_i
```

Comment:
- [ ] It would be better to catch the case where the user enters a bad value (different from `1` and `2`) so that it does not break the app
```rb
 if person_option != "1" && person_option != "2"
      puts "Invalid option"
      return
 end
```